drivers/sensor/isl29125/Kconfig

```diff
@@ -0,0 +1,52 @@
+# ISL29125 Color Light to Digital Converter configuration options
+
+# Copyright(c) 2020 Intive
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig ISL29125
+	bool "ISL29125 Color Light to Digital Converter"
+	depends on I2C
+	help
+		Enable driver for ISL29125 Color Light to Digital Converter.
+
+if ISL29125
+
+choice
+	prompt "Trigger mode"
+	depends on GPIO
+	default ISL29125_TRIGGER_NONE
+	help
+		Specify the type of triggering to be used by the driver.
```

Comment:
This line should be indented two space more than the previous line. (i.e., one tab and two spaces)